cmd/res/devices/devices.yaml

```diff
@@ -0,0 +1,17 @@
+# Pre-defined devices
```

Comment:
The TOML version needs to be removed.
https://github.com/edgexfoundry/device-uart/blob/main/cmd/res/devices/device.uart.toml

Reply:
Removed res/devices/device.uart.toml.